src/components/characters/PCSheetSimple.tsx

```diff
@@ -0,0 +1,169 @@
+import { CSSReset } from "../CSSReset";
+import { ImagePickle } from "../ImagePickle";
+
+import { LogoEditable } from "./LogoEditable";
+import { InputGrid } from "../inputs/InputGrid";
+import { GridField } from "../inputs/GridField";
+import { AsyncTextInput } from "../inputs/AsyncTextInput";
+import { PersonalDetailField } from "./PersonalDetailField";
+import { IndexedAsyncTextInput } from "../inputs/IndexedAsyncTextInput";
+import { Translate } from "../Translate";
+import { MwInjuryStatusWidget } from "./MoribundWorld/MwInjuryStatusWidget";
+import { TrackersArea } from "./TrackersArea";
+import { StatField } from "./StatField";
+import { CombatAbilityDropDown } from "../inputs/CombatAbilityDropDown";
+import { TabContainer } from "../TabContainer";
+import { AbilitiesAreaMW } from "./MoribundWorld/AbilitiesAreaMW";
+import { AbilitiesAreaPlay } from "./AbilitiesAreaPlay";
+import { MwItemArea } from "./MoribundWorld/MwItemArea";
+import { WeaponsArea } from "./Weapons/WeaponsArea";
+import { EquipmentArea } from "./Equipment/EquipmentArea";
+import { NotesArea } from "./NotesArea";
+import { AbilitiesAreaEdit } from "./AbilitiesAreaEdit";
+import { SettingArea } from "./SettingsArea";
+
+
+interface PCSheetProps {
+  actor: InvestigatorActor;
+  foundryApplication: ActorSheet;
+}
+
+export const PCSheetSimple: React.FC<PCSheetProps> = ({
+  actor,
+  foundryApplication,
+}) => {
+  assertGame(game);
+  assertPCActor(actor);
+
+  const updateShortNote = useCallback(
+    async (value: string, index: number) => {
+      await actor.setShortNote(index, value);
+    },
+    [actor],
+  );
+
+  const updateMwHiddenShortNote = useCallback(
+    async (value: string, index: number) => {
+      await actor.setMwHiddenShortNote(index, value);
+    },
+    [actor],
+  );
+
+  const genericOccupation = settings.genericOccupation.get();
+
+  const [occupation, setOccupation] = useState<PersonalDetailItem | undefined>(
+    actor.getOccupations()[0],
+  );
+
+  // some acrobatics here to make sure we update the occupation when it changes
+  // there's no built in hook for "an actor's items changed"
+  useEffect(() => {
+    const callback = (affectedItem: AnyItem) => {
+      if (affectedItem.isOwned && affectedItem.actor?.id === actor.id) {
+        setOccupation(actor.getOccupations()[0]);
+      }
+    };
+    Hooks.on("createItem", callback);
+    Hooks.on("updateItem", callback);
+    Hooks.on("deleteItem", callback);
+    return () => {
+      Hooks.off("createItem", callback);
+      Hooks.off("updateItem", callback);
+      Hooks.off("deleteItem", callback);
+    };
+  }, [actor]);
+
+  const themeName = actor.getSheetThemeName();
+  const theme = useTheme(themeName);
+  const personalDetails = settings.personalDetails.get();
+  const shortHiddenNotesNames = settings.mwHiddenShortNotes.get();
+  const occupationLabel = settings.occupationLabel.get();
+  const stats = settings.pcStats.get();
+
+  return (  <CSSReset
```

Comment:
I haven't had time to do a full review, but I can see the formatting has gone wonky here. You can check formatting and linting locally by running `pnpm eslint:check` and `pnpm format:check` (and the same but with `:fix` to do auto-fixes.)  You should also get yhoiur IDE set up with thr eslint and prettier extensions so you'll get warnings as you type.

Reply:
Also I need to update the CI workflow to run on pull requests!

Reply:
Okay i upgraded my IDE as you suggested (already was using prettier). But i can't see any issues with te layout (except its just the character-portrait until i figured out a method to decice which note-fields should be visible. 

As far as i understand, the "link" option only works on unix-systems as windows does not support ./ to run scripts. Is there a way arround this ? (not very keen to set up a linux system for this).

Reply:
I've just done some science using Windows, and there is hope! 

Linking does not work, but you don't need it. Copy the contents of `build` into your Foundry folder (as you have been doing) and then run `pnpm dev` - it will start up a dev server which will proxy your Foundry instance but serve the files for the INVESTIGATOR system from your project folder.

Reply:
When I said formatting, I meant code formatting: try running `pnpm eslint:fix` and `pnpm format:fix` and see the changes!

Example:

```ts
  return (  <CSSReset
```

will become:

```ts
  return (
    <CSSReset
```

There are also a lot of unused dependencies in `PCSheet.tsx` - all the things that get `import`ed at thje top which are now not used any more.

Reply:
If you have eslint set up in your IDE, it should highlight all the errors, making it a lot easier to fix:
![image](https://github.com/user-attachments/assets/a573c13e-bb8b-4f27-9e57-5df9b37bc690)

Reply:
Okay, i added eslint and tried to get rid of erverything that is not required. Hope this is better. (Sadly i can't make fix to change the formatting as in your example.